.travis.yml

```diff
@@ -33,6 +33,7 @@ install:
     fi
   - pip install autopep8
   - pip install yapf
+  - pip install black
```

Comment:
Should be a conditional on only 3.6 and up. 
For now we can just make it a conditional to only install 3.6 (After 3.7 is out it'll need to be updated)